ctapipe/reco/weighted_axis_minimisation.py

```diff
@@ -0,0 +1,154 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Event reconstruction is nominal and ground system.
+
+The method uses a simple algorithm described in:
+http://arxiv.org/pdf/astro-ph/9904234v1.pdf  (algorithm 6)
+
+This method determines the best fit source and core position by minimising
+the distance between the predicted shower plane and the pixels in the image
+(weighted by pixel amplitude). This is not a very powerful method, but serves
+as an example for more complex and powerful minimisation based methods.
+
+In this case this must be implemented as a class to work properly with the
+iMinuit implementation
+
+ToDo:
+    - Standardise inputs and outputs
+
+"""
+import numpy as np
+import astropy.units as u
+from iminuit import Minuit
+__all__ = [
+    'weighted_axis_minimisation'
+]
+
+
+class weighted_axis_minimisation:
```

Comment:
use CapWords for class names (so `WeightedAxisMinimization`)following the PEP8 standards.

Also, coming soon we will have a nice Configurable base class that can be a parent, and allows for user-parameters to be  added to such a class that automatically can become command-line or conf-file parameters, so this is a good start.